poller/package.json

```
@@ -4,7 +4,6 @@
     "license": "Apache-2.0",
     "author": "Google Inc.",
     "dependencies": {
```

```suggestion
    "dependencies": {
      "@google-cloud/functions-framework": "^2.0.0",
```
Instead of removing the functions-framework, just update its version to 2.0.0